src/azure-cli/azure/cli/command_modules/synapse/manual/commands.py

```diff
@@ -530,10 +534,20 @@ def get_custom_sdk(custom_module, client_factory):
         g.custom_wait_command('wait', 'get_spark_job_definition')
         g.custom_command('update', 'create_or_update_spark_job_definition', supports_no_wait=True)
 
+    with self.command_group('synapse sql-script', synapse_sql_script_sdk,
+                            custom_command_type=get_custom_sdk('artifacts', None)) as g:
+        g.custom_command('list', 'list_sql_scripts')
+        g.custom_show_command('show', 'get_sql_script')
+        g.custom_command('delete', 'delete_sql_script', supports_no_wait=True)
+        g.custom_command('create', 'create_sql_script', supports_no_wait=True)
+        g.custom_wait_command('wait', 'get_sql_script')
+        g.custom_show_command('export', 'export_sql_script')
+        g.custom_command('import', 'create_sql_script', supports_no_wait=True)
```

Comment:
If `import` is just a duplicate of `create`, why keep both `create` and `import`?

Reply:
As an alias of 'create', same in synapse notebook cmdlets